src/kernel/arch/x86_64/gdt/gdt.h

```diff
@@ -0,0 +1,23 @@
+// A purpose for GDT
+
+#pragma once
+
+// Entry of table
+
+struct gdt_entry
+{
+    u16int base_low;
+    u16int limit_low;
+    u8int access_byte;
+    u8int base_middle;
+    u8int base_high;
+}__attribute__((packed));
```

Comment:
You should define a macro in [here](https://github.com/keyboard-slayer/cpcdos-os3/blob/main/src/libs/macro.h), and also include that header.

This make the code cleaner :wink: